contributing.md

```diff
@@ -0,0 +1,125 @@
+updated hyperlinks, better spelling and punctuation) should be merged by the 
+Maintainers without a need for consensus or voting. 
+
+Changes that materially change the meaning of the section should be associated
+with GitHub issues that have to be resolved in the community. 
+
+During periods of **normal mode** development, a section should be at a 
+PROPOSED status until the next working group call (regular or scheduled). 
+
+During periods of **crunch mode** development, this may move as fast as 2 
+business days. 
+* At least two maintainers should still review all PRs, but change-request and 
+discussion should realistically reflect **crunch mode** turnaround. Opening 
+issues for REFINING stage instead of or addition to change requests is 
+encouraged in case of rushed merges.
+* Chairs may ask assignees to check in or progress issues daily
+* Crunch-mode target state is to have PRs merged by the next meeting for any 
+assigned issue.
+
+#### Assigning section content in the REFINING state: 
+
+When the Working Group Participants come to a consensus about the amount and 
+type of content that’s appropriate. While in this state, all substantive changes 
+to the section must be associated with a GitHub issue and must either be 
+approved by a maintainer different from the person who raised the issue, or by 
+consensus among Working Group Participants during a Working Group call. During 
+the REFINING state, the Working Group will request feedback from other 
+decentralized identity-related and -adjacent communities. Ideally, these inputs
+will lead to issues for consensual/deliberative resolution. This merges Further
+validation and improvement of the section can/expected to be done via (code) 
+implementations and testing. 
+
+During periods of **normal mode** development, issue-linked PRs should allow up 
+to 5 business day for review and change requests.  During periods of **crunch 
+mode** development, this window is reduced to 2 business days. At least two 
+maintainers should still review all PRs.
+
+
+#### Assigning section content in the STABLE state: 
+
+when all of the following conditions are met: 
+* There is quality code that demonstrates the concepts it describes.
+* There are no outstanding issues related to that section that the Working Group 
+Participants consider substantive.
+* The Working Group has at least 2 conformant implementations, at least 1 of
+which originated outside of the Working Group 
+
+When consensus has been arrived at that the above conditions have been reached, 
+the section enters a final 2-week comment period has elapsed without any 
+substantive objections to the LAST-COMMENTS status. The start of this period, 
+intended for "last comments," should be announced on the Working Group mailing 
+list. If substantial discussion or change happens, this LAST-COMMENTS phase can
+be extended 1 additional week at the discretion of the chairs.
```

Comment:
```suggestion
the section enters a final 2-week comment period if no substantive
objections are raised against the LAST-COMMENTS status. The start
of this period, intended for "last comments," should be announced
on the Working Group mailing list. If substantial discussion or change
happens, this LAST-COMMENTS phase can be extended 1 additional
week at the discretion of the chairs.
```

Not sure if this wording captures the intended meaning, but the original text wasn't clear to me.